ovos_plugin_manager/language.py

```diff
@@ -99,6 +99,12 @@ def get_lang_detect_module_configs(module_name: str):
     return load_plugin_configs(module_name, PluginConfigTypes.LANG_DETECT)
 
 
+_default_lang_detect_plugin = "ovos-lang-detect-ngram-lm"
+_fallback_lang_detect_plugin = "libretranslate_detection_plug"
+_default_translate_plugin = "libretranslate_plug"
```

Comment:
this should be updated to the server plugin, we already provide a public server list for NLLB